src/test/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/CloudStackCloudUtilsTest.java

```diff
@@ -0,0 +1,78 @@
+package cloud.fogbow.ras.core.plugins.interoperability.cloudstack;
+
+import cloud.fogbow.common.exceptions.FogbowException;
+import cloud.fogbow.common.models.CloudStackUser;
+import cloud.fogbow.common.util.connectivity.cloud.cloudstack.CloudStackHttpClient;
+import cloud.fogbow.ras.core.TestUtils;
+import org.apache.http.client.HttpResponseException;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mockito;
+
+public class CloudStackCloudUtilsTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    // test case: When calling the doGet method with a right parameter,
+    // it must verify if It returns the subnetInfo expected.
+    @Test
+    public void testDoGetSuccessfully() throws FogbowException, HttpResponseException {
+        // set up
+        String responseExpected = "responseExpected";
+        String url = "";
+
+        CloudStackUser cloudStackUser = CloudstackTestUtils.CLOUD_STACK_USER;
+        CloudStackHttpClient client = Mockito.mock(CloudStackHttpClient.class);
+        Mockito.when(client.doGetRequest(Mockito.eq(url), Mockito.eq(cloudStackUser))).
+                thenReturn(responseExpected);
+
+        // exercise
+        String response = CloudStackCloudUtils.doGet(client, url, cloudStackUser);
+
+        // verify
+        Assert.assertEquals(responseExpected, response);
+        Mockito.verify(client, Mockito.times(TestUtils.RUN_ONCE)).
+                doGetRequest(Mockito.eq(url), Mockito.eq(cloudStackUser));
+    }
+
+    // test case: When calling the doRequest method and
+    // it occurs an HttpResponseException, it must verify if It returns a HttpResponseException.
+    @Test
+    public void testDoGetFail() throws FogbowException, HttpResponseException {
+        // set up
+        String url = "";
+        CloudStackUser cloudStackUser = CloudstackTestUtils.CLOUD_STACK_USER;
+        CloudStackHttpClient client = Mockito.mock(CloudStackHttpClient.class);
+        Mockito.when(client.doGetRequest(Mockito.eq(url), Mockito.eq(cloudStackUser))).
+                thenThrow(CloudstackTestUtils.createBadRequestHttpResponse());
+
+        this.expectedException.expect(HttpResponseException.class);
```

Comment:
please insert //verify comment

Reply:
check the same thing in the others tests

Reply:
done